requirements.txt

```diff
@@ -0,0 +1,7 @@
+Django==2.0.5
```

Comment:
This app may be used in projects of varying Django versions. It is up to the project to specify any specific third-party versions it requires; an app should be liberal for whatever versions it may accept.
So the Django line in `setup.py` should be changed to specify 1.11 as a minimum, but I don't think a requirements.txt file is necessary.